core/storage/base_model/gae_models.py

```
@@ -2247,12 +2248,12 @@ def _get_filtered_query(
         """
         query = cls.query()
         # Ignoring deleted threads as they are not relevant for operations.
-        query = query.filter(cls.deleted.IN([False]))
+        query = query.filter(cls.deleted == False)
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `False`; use `not cls.deleted:` for false checks

Replace with `not cls.deleted`

(E712)